src/docs/get-started/flutter-for/android-devs.md

```diff
@@ -2082,23 +2114,20 @@ class _SampleAppPageState extends State<SampleAppPage> {
             });
           },
           decoration: InputDecoration(
-            hintText: "This is a hint",
+            hintText: 'This is a hint',
             errorText: _getErrorText(),
           ),
         ),
       ),
     );
   }
 
-  _getErrorText() {
+  String _getErrorText() {
     return _errorText;
   }
 
   bool isEmail(String em) {
-    String emailRegexp =
-        r'^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$';
-
-    RegExp regExp = RegExp(emailRegexp);
+    RegExp regExp = RegExp(r'^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$');
```

Comment:
For some reason, this change is breaking the build. I wondered if there were any single quotes that needed to be escaped, but there isn't. Can you fix this? I need the build to be happy before I can merge.

Reply:
I don't know why build failed. I rolled it back to the existing code and I'm very new to Github in pulling request, so let me know if there's something more I need to do.

Reply:
It might be a fluke. That happens sometimes, esp with GitHub Actions. (We used to use Travis, but it was MUCH slower.) Anyway, I can probably land this after the tests run again.

Reply:
Do I need to run the tests? Sorry if this all sound stupid because as I said this is my first pull request.

Reply:
I'm going to go ahead and land it. I could have re-run the checks, but I'm not sure if external people can do it. You attempted a very ambitious PR for your first.  :D